lib/StatusIndicator/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StatusIndicator = props => 
+  <span>
```

Comment:
It's be good if we add a base class to this for styling purposes e.g. `status-indicator`.